### ISSUE TITLE
[DOC] Remoting.html doesn't exist

### ISSUE THREAD

**Issue by ahmelsayed** [fsprojects/Cricket#53]
### Description

Documentation page at http://fsprojects.github.io/Cricket/remoting.html gives a 404

### Repro steps

1. click on http://fsprojects.github.io/Cricket/remoting.html :)

### Expected behavior

HTTP 200 OK

### Actual behavior

HTTP 404 Not Found

### Known workarounds

It looks like it was removed in e89e3160adf947b957058d33b4b5327df7527366

**Comment by colinbull** [comment]
Yeah, remoting has been removed from cricket. I must have missed this. I'll update the docs.
